crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -284,23 +285,25 @@ impl BuildSpecCommand {
 
 		// Generate genesis artifacts.
 		if self.genesis_code {
-			cli.info("Generating genesis code...")?;
+			spinner.set_message("Generating genesis code...");
 			let wasm_file_name = format!("para-{}.wasm", para_id);
 			let wasm_file = export_wasm_file(&binary_path, &raw_chain_spec, &wasm_file_name)?;
 			generated_files
 				.push(format!("WebAssembly runtime file exported at: {}", wasm_file.display()));
 		}
 
 		if self.genesis_state {
-			cli.info("Generating genesis state...")?;
+			spinner.set_message("Generating genesis state...");
 			let genesis_file_name = format!("para-{}-genesis-state", para_id);
 			let genesis_state_file =
 				generate_genesis_state_file(&binary_path, &raw_chain_spec, &genesis_file_name)?;
 			generated_files
 				.push(format!("Genesis State file exported at: {}", genesis_state_file.display()));
 		}
 
-		cli.intro("Building your chain spec".to_string())?;
+		spinner.stop("Done.");
```

Comment:
If using a single spinner, would be better as 'Chain specification built successfully.' or something.